pykechain/client.py

```diff
@@ -1024,6 +1024,18 @@ def widgets(
 
         return [Widget.create(json=json, client=self) for json in response.json()['results']]
 
+    def widget(self, *args, **kwargs) -> Widget:
+        """
+        Widget of an activity.
+
+        :param pk: (optional) the uuid of the widget.
+        :type pk: basestring or None
```

Comment:
Add those parameters as function arguments (even if they are optional). Otherwise, there will be complaints that they are unexpected parameters in the docstring